packages/components/pages/f-contact-preferences/test-utils/component-objects/f-contactPreferences.component.js

```diff
@@ -1,14 +1,24 @@
 /* eslint-disable class-methods-use-this */
 const Page = require('@justeat/f-wdio-utils/src/page.object');
+const {
+    COMPONENT,
+    ERROR_PAGE,
+    SUBMIT_BUTTON,
+    CHECKBOXES
+} = require('./selectors');
 
 module.exports = class ContactPreferences extends Page {
     constructor () {
         super('page', 'contact-preferences-component');
     }
 
-    get component () {
-        return $('[data-test-id="contactPreferences"]');
-    }
+    get component () { return $(COMPONENT); }
+
+    get errorPage () { return $(ERROR_PAGE); }
+
+    get submitButton () { return $(SUBMIT_BUTTON); }
+
+    get newsEmailPreference () { return $(CHECKBOXES.news.email); }
```

Comment:
Add new selector for newsSmsPreference

Reply:
Whilst you won't be utilising it here, it's good to have for any future CoreWeb tests 👍🏼